pyingest/parsers/jats.py

```diff
@@ -388,6 +407,16 @@ def parse(self, fp, **kwargs):
                         base_metadata['pubdate']
                     except Exception, err:
                         base_metadata['pubdate'] = pubdate
+            try:
+                if (b == 'open-access'):
+                    try:
```

Comment:
don't understand this bit of the code.  Should we not create base_metadata['properties'] as a dictionary if it doesn't already exist so we can set the open access flag later on?